deployment/fedora-package-x64/clean.sh

```diff
@@ -4,4 +4,15 @@ source ../common.build.sh
 
 VERSION=`get_version ../..`
 
-clean_jellyfin ../.. Release `pwd`/dist/jellyfin_${VERSION}
```

Comment:
The `clean_jellyfin` function should be doing all this, no? It's working fine for the Debian package.

Reply:
The `clean_jellyfin` uses `dotnet clean` which is not needed since the package is build inside a docker container.